File: Bon_plans_desktop_app/src/tn/esprit/bonplans/entity/Reservation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tn.esprit.bonplans.entity;

import java.util.Date;

/**
 *
 * @author SadfiAmine
 */
public class Reservation {
    private int idReservation;
    private Date date;
    private String UrlBonRes; 
    private int nbplace;
    private int status;
    //
    private int idClient;
    private int idPlan;

    public Reservation(Date date) {
        this.date = date;
    }

    public Reservation(int idReservation, Date date,String UrlBonRes,int nbplace) {
        this.idReservation = idReservation;
        this.date = date;
        this.UrlBonRes=UrlBonRes;
        this.nbplace=nbplace;
    }

    public Reservation(Date date, String UrlBonRes, int idClient, int idPlan,int nbplace) {
        this.date = date;
        this.UrlBonRes = UrlBonRes;
        this.idClient = idClient;
        this.idPlan = idPlan;
        this.nbplace=nbplace;
    }
    
   
    public Reservation() {
    }

    public int getIdReservation() {
        return idReservation;
    }

    public void setIdReservation(int idReservation) {
        this.idReservation = idReservation;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getIdClient() {
        return idClient;
    }

    public void setIdClient(int idClient) {
        this.idClient = idClient;
    }

    public int getIdPlan() {
        return idPlan;
    }

    public void setIdPlan(int idPlan) {
        this.idPlan = idPlan;
    }

    public String getUrlBonRes() {
        return UrlBonRes;
    }

    public void setUrlBonRes(String UrlBonRes) {
        this.UrlBonRes = UrlBonRes;
    }

    public int getNbplace() {
        return nbplace;
    }

    public void setNbplace(int nbplace) {
        this.nbplace = nbplace;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }
    
    

    @Override
    public String toString() {
        return "Reservation{" + "idReservation=" + idReservation + ", date=" + date + ", idClient=" + idClient + ", idPlan=" + idPlan + '}';
    }
    
    
    
}

File: www/Bon_plans_web_app/Entity/Reservation.php
<?php
/**
 * Created by PhpStorm.
 * User: SadfiAmine
 * Date: 15/04/2018
 * Time: 12:53
 */

class Reservation
{
    private $idReservation;
    private $date;
    private $UrlBonRes;
        //
    private $idClient;
    private $idPlan;

    public function __construct()
    {
    }
    public function __construct1($idReservation,$date,$UrlBonRes)
    {   $this->$idReservation=$idReservation;
        $this->$date=$date;
        $this->$UrlBonRes=$UrlBonRes;
    }
    public function __construct2($date,$UrlBonRes,$idClient,$idPlan)
    {
        $this->$date=$date;
        $this->$UrlBonRes=$UrlBonRes;
        $this->$idClient=$idClient;
        $this->$idPlan=$idPlan;
    }
    public function __construct3($date,$UrlBonRes)
    {
        $this->$date=$date;
        $this->$UrlBonRes=$UrlBonRes;
    }

    /**
     * @return mixed
     */
    public function getIdPlan()
    {
        return $this->idPlan;
    }

    /**
     * @param mixed $idPlan
     */
    public function setIdPlan($idPlan)
    {
        $this->idPlan = $idPlan;
    }

    /**
     * @return mixed
     */
    public function getIdClient()
    {
        return $this->idClient;
    }

    /**
     * @param mixed $idClient
     */
    public function setIdClient($idClient)
    {
        $this->idClient = $idClient;
    }

    /**
     * @return mixed
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * @param mixed $date
     */
    public function setDate($date)
    {
        $this->date = $date;
    }

    /**
     * @return mixed
     */
    public function getIdReservation()
    {
        return $this->idReservation;
    }

    /**
     * @param mixed $idReservation
     */
    public function setIdReservation($idReservation)
    {
        $this->idReservation = $idReservation;
    }

    /**
     * @return mixed
     */
    public function getUrlBonRes()
    {
        return $this->UrlBonRes;
    }

    /**
     * @param mixed $UrlBonRes
     */
    public function setUrlBonRes($UrlBonRes)
    {
        $this->UrlBonRes = $UrlBonRes;
    }


}